src/App.tsx

```diff
@@ -1,12 +1,82 @@
-import { Routes, Route } from "react-router-dom";
-import Home from "./pages/Home";
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+import { Routes, Route, Link } from "react-router-dom";
+import Home from "pages/Home";
+import Menu from "pages/Menu";
+import MenuDetail from "pages/MenuDetail";
+import Profile from "pages/Profile";
+import Order from "pages/Order";
+import OrderDetail from "pages/OrderDetail";
 
+//아래 Route외 html Layout 컴포넌트로 변경 예정
 function App() {
   return (
-    <Routes>
-      <Route path="/" element={<Home />} />
-    </Routes>
+    <div css={_container}>
+      <aside css={_sideMenu}>
+        <h1>
+          <Link to="/" aria-label="Home">
+            LOGO 자리
+          </Link>
+        </h1>
+        <nav aria-label="Sidebar navigation">
+          <ul>
+            <li>
+              <Link to="/dashboard">Dashboard</Link>
+            </li>
+            <li>
+              <Link to="order">Order</Link>
+            </li>
+            <li>
+              <Link to="menu">Menu</Link>
+            </li>
+            <li>
+              <Link to="review">Review</Link>
+            </li>
+          </ul>
+        </nav>
+      </aside>
+      <div css={_wrapper}>
+        <header css={_header}>
+          <div>
+            <form role="search">
+              <label htmlFor="search">Search</label>
+              <input id="search" type="search" placeholder="Search..." />
+            </form>
+          </div>
+        </header>
+        <main>
+          <Routes>
+            <Route path="/" element={<Home role="store-owner" />} />
+            <Route path="/menu" element={<Menu />} />
```

Comment:
path 속성에 들어가는 routing 경로들을 상수화해서 관리하면 어떨까 싶어요!
추후에 각 페이지에서도 해당 경로를 사용할탠데 하나의 상수로 관리하면 변경에 대응하기 좋다고 생각해서요!
```suggestion
            <Route path={ROUTES.MENU} element={<Menu />} />
```

Reply:
좋은 생각인거 같아요 승완님!!

Reply:
@Seung-wan 네 좋은 생각입니다!